website/views/shared/profiles/stats/costume.jade

```diff
@@ -0,0 +1,6 @@
+unless mobile
```

Comment:
Ok then!
On Sat, Jul 9, 2016 at 6:36 AM Blade Barringer <notifications@github.com>
wrote:

> In website/views/shared/profiles/stats/costume.jade
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r70166022>:
>
> > @@ -0,0 +1,6 @@
> > +unless mobile
>
> Not necessary. This was for the phonegap apps which aren't used anymore.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/717de41e73c30ce18a8374f124fb2633e77da48a#r70166022>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALz_8wQrMSRR70UKKA2lT-qmGjzm3bB2ks5qT4evgaJpZM4JIjd_>
> .
>